spec/services/idv/profile_step_spec.rb

```diff
@@ -13,137 +12,85 @@
       address1: '123 Main St',
       address2: '',
       city: 'Somewhere',
-      state: 'KS',
+      state: 'VA',
       zipcode: '66044',
+      state_id_number: '123abc',
+      state_id_type: 'drivers_license',
     }
   end
 
-  def build_step(params, vendor_validator_result)
-    idv_session.params.merge!(params)
-    idv_session.applicant = idv_session.vendor_params
-
-    described_class.new(
-      idv_form_params: params,
-      vendor_validator_result: vendor_validator_result,
-      idv_session: idv_session
-    )
-  end
+  subject { described_class.new(idv_session: idv_session) }
 
   describe '#submit' do
     it 'succeeds with good params' do
-      messages = ['Everything looks good']
```

Comment:
Are we not using these messages anymore?

Reply:
The proofer gems will still return them in some instances. The change here is that we were stubbing the responses from the proofer gem. I removed the stubs and had the tests talk to the mock proofers. The mock proofers do not return messages in their response.